Houdini/src/Controller/HoudiniController.php

```diff
@@ -173,4 +182,29 @@ public function identify(Request $request)
             return new Response($e->getMessage(), 500);
         }
     }
+
+    /**
+     * imagemagick will sometimes leave tmp files behind, this will clean them up.
+     * Be careful about when you call this, else you could delete work that's in
+     * progress.
+     */
+     protected function cleanupTmpFiles() {
+
+        // If there's are magick-* or php* files in /tmp, clean them up.
+        // We could just do this, but for now let's add some logging.
+        //   array_map('unlink', glob('/tmp/magick-*'));
+        //   array_map('unlink', glob('/tmp/php*'));
+        try {
+            foreach (glob('/tmp/magick-*') as $filename) {
+                $this->log->info("removing file $filename");
+                unlink($filename);
+            }
+            foreach (glob('/tmp/php*') as $filename) {
```

Comment:
This part that cleans up php tmp files might not be necessary. should I leave this in?